src/main/java/org/usfirst/frc4904/robot/RobotMap.java

```diff
@@ -70,5 +114,38 @@ public RobotMap() {
         HumanInput.Driver.xbox = new CustomXbox(Port.HumanInput.xboxController);
 		HumanInput.Operator.joystick = new CustomJoystick(Port.HumanInput.joystick);
 
+        /* Drive Train */
+        // TalonFX
+        CANTalonFX leftWheelATalon = new CANTalonFX(Port.CANMotor.LEFT_DRIVE_A);
+        CANTalonFX leftWheelBTalon = new CANTalonFX(Port.CANMotor.LEFT_DRIVE_B);
+        CANTalonFX rightWheelATalon = new CANTalonFX(Port.CANMotor.RIGHT_DRIVE_A);
+        CANTalonFX rightWheelBTalon = new CANTalonFX(Port.CANMotor.RIGHT_DRIVE_B);
+
+        // Wheels
+        Component.rightWheelA = new Motor("rightWheelA", false, rightWheelATalon);
+        Component.rightWheelB = new Motor("rightWheelB", false, rightWheelBTalon);
+        Component.leftWheelA = new Motor("leftWheelA", true, leftWheelATalon);
+        Component.leftWheelB = new Motor("leftWheelB", true, leftWheelBTalon);
+
+        // Wheel Encoders
+        Component.leftWheelCANCoder = new CustomCANCoder(Port.CAN.LEFT_WHEEL_ENCODER,
+                Metrics.Chassis.METERS_PER_TICK);
+        Component.rightWheelCANCoder = new CustomCANCoder(Port.CAN.RIGHT_WHEEL_ENCODER,
+                Metrics.Chassis.METERS_PER_TICK);
+
+        Component.leftWheelTalonEncoder = new CANTalonEncoder("leftWheel", leftWheelATalon, true,
+                Metrics.Encoders.TalonEncoders.REVOLUTIONS_PER_TICK, CustomPIDSourceType.kDisplacement,
+                FeedbackDevice.IntegratedSensor);
+        Component.rightWheelTalonEncoder = new CANTalonEncoder("rightWheel", rightWheelATalon, true,
+                Metrics.Encoders.TalonEncoders.REVOLUTIONS_PER_TICK, CustomPIDSourceType.kDisplacement,
+                FeedbackDevice.IntegratedSensor);
+
+        Component.chassisTalonEncoders = new EncoderPair(Component.leftWheelTalonEncoder, Component.rightWheelTalonEncoder);
+        Component.chassisCANCoders = new EncoderPair(Component.leftWheelCANCoder, Component.rightWheelCANCoder);
+
+        // General Chassis
+        Component.chassis = new TankDrive("2022-Chassis", Component.leftWheelA, Component.leftWheelB,
+                Component.rightWheelA, Component.rightWheelB);
+        Component.chassis.setDefaultCommand(new ChassisMove(Component.chassis, new NathanGain()));
     }
 }
```

Comment:
Sorry, Brandon. There's no newline at the end of this file; I cannot in good conscience approve this pull request.